### DISCUSSION TITLE
Close Menu on outside-click

### DISCUSSION THREAD

**Discussion by tomsoja978** [primefaces/community#1008] [PrimeReact]
I am using Menu:

https://primereact.org/menu/

Once the menu has been opened, i can apparently only closing it by making the parent component rerender or clicking on the cog-button (which opens the menu) again. I wish i could simply close the menu by clicking somewhere outside of it. How can i achieve that?

**Comment by melloware** [comment]
Not sure what you are referring to but: https://primereact.org/menu/#popup

![image](https://github.com/primefaces/community/assets/4399574/079c6821-329e-4ea2-ab07-d85813e1a9f6)

IF I click anywhere outside the menu it closes.
